docs/modules/deployment/pages/core/docker/selinux.adoc

```diff
@@ -0,0 +1,3 @@
+
+Documentation missing?
+Click *Edit This Page* in the top-right corner to contribute.
```

Comment:
Docker and SELinux are a different story. We can skip this here for now. We would need some more investigation in a separate JIRA ticket.

Reply:
Agree that this probably isn't necessary on Docker.